src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -1066,7 +1066,7 @@ export class EditDataGridPanel extends GridParentComponent {
 			cellClasses += ' missing-value';
 		}
 		else if (Services.DBCellValue.isDBCellValue(value)) {
-			valueToDisplay = (value.displayValue + '').replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;');
+			valueToDisplay = (value.displayValue + '');
```

Comment:
this is unnecessary now because it's being handled somewhere else?

Reply:
It isn't handled here, are there any instances that might require escaping characters?

Reply:
@smartguest thanks.  In general it is good to root cause the exact reason by the application behavior.  i.e., if this is no longer needed we should understand why not and how character escaping is being handled now.  This helps understand if there are potential side-effects from a change.